src/web/components/StickyBottomBanner/ReaderRevenueBanner.tsx

```diff
@@ -208,7 +208,7 @@ export const ReaderRevenueBanner = ({ meta, module }: Props) => {
             // The css here is necessary to put the container div in view, so that we can track the view
```

Comment:
Are we tracking the view still, given this comment?

Reply:
good question - I've checked this still works and added a screenshot of the ophan request